Neos.Fusion/Classes/FusionObjects/ComponentImplementation.php

```diff
@@ -71,6 +75,17 @@ protected function getProps(array $context): \ArrayAccess
         return $props;
     }
 
+    protected function getPrivateProps(array $context): ?\ArrayAccess
```

Comment:
maybe `getPrivates` ... `getPrivateProps` sounds a bit weird

Reply:
getPrivates could also be weird though

Reply:
and make it `private` :P

i like the current name

Reply:
Please not [getPrivates](https://www.linguee.de/deutsch-englisch/search?source=englisch&query=privates) 😄